source/adapters/cuda/CMakeLists.txt

```
@@ -73,6 +73,10 @@ else()
   message(WARNING "CUDA adapter USM pools are disabled, set UMF_ENABLE_POOL_TRACKING to enable them")
 endif()
 
+if (CUDA_cupti_LIBRARY)
+  target_compile_definitions("ur_adapter_cuda" PRIVATE "-DCUPTI_LIB_PATH=\"${CUDA_cupti_LIBRARY}\"")
```

I don't think need the `-D` or the outer quotes when using `target_compile_definitions`

```suggestion
  target_compile_definitions(ur_adapter_cuda PRIVATE CUPTI_LIB_PATH="${CUDA_cupti_LIBRARY}")
```

---

Good catch, that looks a bit simpler/cleaner now.